backend/src/datasources/user.datasource.ts

```diff
@@ -0,0 +1,33 @@
+import User from '../entities/user.entity';
+import { DataSource } from 'apollo-datasource';
+import { users } from '../seed-data' 
+
+export default class UserDatasource extends DataSource{
+    private users: User[];
+
+    constructor() {
+        super();
+
+        this.users = users;
+    }
```

Comment:
```suggestion
    constructor(initialUsers = []) {
        super();

        this.users = initialUsers;
    }
```
gives you much more flexibility for e.g. testing.

Only for development you could create some initial users right before the server starts.

```js
import { users } from '../seed-data' 

const userDatasource = UserDatasource(users)
```